src/networking/VoiceUDPSocket.ts

```diff
@@ -74,16 +77,27 @@ export class VoiceUDPSocket extends EventEmitter {
 	}
 }
 
+const IPV4_REGEX = new RegExp(/^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])(\.(?!$)|$)){4}$/);
```

Comment:
```suggestion
const IPV4_REGEX = /^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])(\.(?!$)|$)){4}$/;
```

Reply:
Alternatively, [`net.isIPv4`](https://nodejs.org/dist/latest-v15.x/docs/api/net.html#net_net_isipv4_input) can also be used.

Reply:
Rather the latter.